src/status_im/ui/screens/wallet/main/views.cljs

```diff
@@ -7,102 +7,138 @@
             [status-im.components.drawer.view :as drawer]
             [status-im.components.list.views :as list]
             [status-im.components.react :as rn]
+            [status-im.components.styles :as st]
             [status-im.components.icons.vector-icons :as vi]
             [status-im.components.toolbar-new.view :as toolbar]
             [status-im.components.toolbar-new.actions :as act]
             [status-im.i18n :as i18n]
+            [status-im.react-native.resources :as resources]
             [status-im.utils.config :as config]
             [status-im.utils.platform :as platform]
             [status-im.utils.utils :as utils]
-            [status-im.ui.screens.wallet.main.styles :as st]
+            [status-im.ui.screens.wallet.main.styles :as wst]
             [status-im.utils.money :as money]))
 
 (defn- show-not-implemented! []
   (utils/show-popup "TODO" "Not implemented yet!"))
 
 (defn toolbar-title []
   [rn/touchable-highlight {:on-press #(rf/dispatch [:navigate-to :wallet-list])
-                           :style st/toolbar-title-container}
-   [rn/view {:style st/toolbar-title-inner-container}
-    [rn/text {:style st/toolbar-title-text
+                           :style wst/toolbar-title-container}
+   [rn/view {:style wst/toolbar-title-inner-container}
+    [rn/text {:style wst/toolbar-title-text
               :font  :toolbar-title}
      (i18n/label :t/main-wallet)]
     [vi/icon
      :icons/dropdown
-     {:container-style st/toolbar-title-icon
+     {:container-style wst/toolbar-title-icon
       :color :white}]]])
 
 (defn toolbar-buttons []
-  [rn/view {:style st/toolbar-buttons-container}
-   [vi/touchable-icon :icons/dots_vertical {:color :white} show-not-implemented!]
-   [vi/touchable-icon :icons/qr {:color :white} show-not-implemented!]])
+  [rn/view {:style wst/toolbar-buttons-container}
```

Comment:
wst says nothing, better to use wallet-styles

Reply:
ok

Reply:
thanks